README.md

```diff
@@ -228,6 +229,39 @@ You can define and use your own custom rules using the plugin system. See [plugi
 
 It is possible to share a config (`extends`) or plugin (custom rules) across projects. See [ember-template-lint-plugin-peopleconnect](https://github.com/peopleconnectus/ember-template-lint-plugin-peopleconnect) for an example.
 
+## Semantic Versioning Policy
+
+`ember-template-lint` follows [semantic versioning](http://semver.org/).
+
+## Patch Release
+
+A patch release should not break your lint build.
+
+* A bug fix in a rule that results in it reporting fewer errors.
+* Improvements to documentation.
+* Non-user-facing changes such as refactoring code, adding, deleting, or modifying tests, and increasing test coverage.
+* Re-releasing after a failed release (i.e., publishing a release that doesn't work for anyone).
+
+### Minor Release
+
+A minor release may break your lint build in some cases.
+
+* A bug fix in a rule that results in it reporting more errors.
+* A new rule is created.
+* A new option to an existing rule is created.
+* An existing rule is deprecated.
+
+### Major Release
+
+A major release is likely to break your lint build.
+
+* A support for old Node version is dropped.
+* A support for old ember-template-lint version is dropped.
+* An existing rule is changed in it reporting more errors.
+* An existing rule is removed.
+* An existing option of a rule is removed.
+* An existing config is updated.
```

Comment:
seems quite verbose to me to be included in the `README.md` file. maybe we could move the majority of it into a `VERSIONING.md` or `docs/versioning.md` and only keep a reference to that file in the README?

Reply:
Sure! 

Just to make sure, is there a less verbose way to say this?

Reply:
just to clarify, I think the content itself is fine, just too long to be included in the README itself. :)

Reply:
Ya, that sounds good to me. 

@MelSumner - Any objections?

Reply:
No objections! Will do!